modules/apps/frontend-js/frontend-js-a11y-web/src/main/resources/META-INF/resources/Violation.tsx

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import './Violation.scss';
+
+import ClayList from '@clayui/list';
+import ClayPanel from '@clayui/panel';
+import React from 'react';
+
+import PanelNavigator from './PanelNavigator';
+import Rule from './Rule';
+
+import type {Result} from 'axe-core';
+
+declare var Liferay: {
+	Language: {
+		get(value: string): string;
+	};
+};
```

Comment:
Hey @diegonvs if you just declare in `index.ts` as global it should be enough instead of declaring in each file?

Reply:
Makes sense. I'll try it out!

Reply:
Needed to be declared as a global inside module's `index.ts` to make it work
```ts
declare global {
	var Liferay: {
		Language: {
			get(value: string): string;
		};
		Util: {
			sub(...value: string[]): string;
		};
	};
}
```